core/arch/arm/plat-totalcompute/tc0_spmc_pm.c

```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: BSD-2-Clause
+/* SPDX-License-Identifier: BSD-2-Clause */
```

Comment:
The `//` style of comment is actually preferred in .c files for SPDX comments (.h files must use the `/* */` style though).
This is also the Linux kernel style.

Reply:
Fixed. Used // style of comment in .c file for SPDX comments.